webhook_server/tests/test_graphql_wrappers.py

```
@@ -0,0 +1,243 @@
+    assert wrapper.head.repo.name == "repo-name"
+
+
+def test_ref_wrapper_without_repository_raises_error():
+    """Test RefWrapper without repository info raises AttributeError."""
+    ref_data = {"name": "main", "target": {"oid": "abc123"}}
+    from webhook_server.libs.graphql.graphql_wrappers import RefWrapper
+
+    ref = RefWrapper(ref_data)
+    try:
+        _ = ref.repo
+        assert False, "Expected AttributeError"
```

Ruff (0.14.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)